front/app/components/UI/Error/index.tsx

```diff
@@ -172,6 +172,21 @@ export type TFieldName =
   | 'category_name'
   | 'nav_bar_item_title_multiloc';
 
+export const findErrorMessage = (
+  fieldName: TFieldName | undefined,
```

Comment:
Is it necessary to have `undefined` as a possible type here? I feel like we shouldn't be calling this function if we don't have a `fieldName`. I also don't see any place in the code where this is used where it could be `undefined` (or am I missing something?). 

Also, it would be great if this function would just always return an error message- saves some more type checks in places where it's being used, e.g. in `front/app/components/UI/RHFFeedback/index.tsx` (line 57). I think there should always be an error message for every error, right (so `messages[error]` will always be truthy)?

Reply:
Good points! 👍